lib/index.js

```diff
@@ -0,0 +1,103 @@
+const fs = require('fs-extra');
+const { config } = require('./config/config');
+const { getTestResults } = require('./test-handler/case');
+const { getTestResults: getTestResultsByDescribe } = require('./test-handler/describe');
+const { api } = require('./api');
+
+const repostFile = fs.readJsonSync(config.reportsFile.JSON);
```

Comment:
What do you think about adding the ability to import another config in the library itself? This is necessary for the development mode.